pkg/server/serverpb/status.proto

```diff
@@ -629,6 +629,7 @@ message StatementsResponse {
     bool failed = 4;
     int32 node_id = 5 [ (gogoproto.customname) = "NodeID",
                         (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/roachpb.NodeID"];
+    bool opt = 6;
```

Comment:
Oh this is too bad, I bet the other proto now has `opt = 5`, doesn't it?  Let's try to unfork these maybe in the future?

Reply:
can we be a little more descriptive here? `optimizer_used` or something?